.github/workflows/build.yml

```diff
@@ -62,3 +62,16 @@ jobs:
           mvn -B -ntp versions:update-parent -DallowSnapshots=true -N
           git diff pom.xml
           mvn -B -ntp install -Djava8.home=${{env.JAVA_HOME_8_X64}} -Djava11.home=${{env.JAVA_HOME_11_X64}}
+
+      - name: Check out WildFly Maven Plugin
+        uses: actions/checkout@v2
+        with:
+          repository: wildfly/wildfly-maven-plugin
+          path: wildfly-maven-plugin
+
+      - name: Test WildFly Maven Plugin with updated parent
+        run: |
+          cd wildfly-maven-plugin
+          mvn -B -ntp versions:update-parent -DallowSnapshots=true -N
+          git diff pom.xml
+          mvn -B -ntp install -Djava8.home=${{env.JAVA_HOME_8_X64}} -Djava11.home=${{env.JAVA_HOME_11_X64}}
```

Comment:
Will this work? The wildfly-maven-plugin requires Java 11 and uses the `org.wildfly.tools:wildfly-parent` https://github.com/wildfly/wildfly-maven-plugin/blob/61bfbe9dd704afe92ad07e686e3425f45ec88085/pom.xml#L27-L31.

Reply:
Yes, specifying a `java8.home` doesn't cause the profile to run; it only runs if the control file is present. The idea is that if we cut & paste the entry for other projects, we don't lose anything.

That said, we've since dropped Java 8 support from the POM in #207 so we could drop the install action for Java 8 and these properties without any side-effect.